src/main/scala/jsy/lab4/Lab4Like.scala

```diff
@@ -83,8 +83,7 @@ trait Lab4Like { a: JsyApplication =>
   def iterateStep(s: String): Expr = iterateStep(parse(s))
 
   // Interface for main
-  def processFile(file: java.io.File) {
-    val debug = true
```

Comment:
Oh, this could be what's changing the expected `.ans` output. We want to remove this? What did we have in Lab 3?

Reply:
Let's use the pppl-labdev version. I'm considering this resolved.